### PR TITLE
Create SolarWinds Orion Dashboard Repository

### PR DESCRIPTION
**Pull request by vinbab**

### CODE REVIEW

**Review by john-knowles — APPROVED** [review]
Great first contribution @vinbab

File: dashboards/ds-solarwinds-webapi/readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# SolarWinds Orion Nodes `API`
+
+This dashboard gives an overview of nodes alongside summary of heatlh from SolarWinds Orion using the powerful Web API and dashboard sharing capabilities of SquaredUp. 
+
+## Requirements
+
+- Copy the images repository on the SquaredUp IIS folder (or individually if the images repository already exists)
```
Comment (adamkinniburgh) [inline dashboards/ds-solarwinds-webapi/readme.md:7]: Maybe mention that the folder `images` needs to exist specifically in the IIS root i.e. `C:\inetpub\wwwroot` and the structure of `\health-icons\` and `\logos\` should be maintained (or the dashboard JSON updated to suit).

File: dashboards/ds-solarwinds-webapi/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- Create a WebAPI (Basic Auth) provider under Settings > Integrations. Name it exactly “Solarwinds” (if you name it differently, you will need to select the provider manually for each tile on the dashboard). Fill in all the required details to connect to your own SolarWinds environment (url, username, password, ignore invalid ssl, etc).
+- Create a new dashboard, select the <> on the top right and copy the content of the dashboard.json file and click Apply Changes
+
```
Comment (adamkinniburgh) [inline dashboards/ds-solarwinds-webapi/readme.md:10]: Maybe call out updating the `rowLink` here too

**Review by adamkinniburgh — CHANGES_REQUESTED** [review]
Harsh reviewer :)

File: dashboards/ds-solarwinds-webapi/dashboard.json [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+                            "url": "query",
+                            "data": [
+                              {
+                                "key": "query",
+                                "value": "SELECT NodeID, Status\nFROM Orion.Nodes"
+                              }
+                            ],
+                            "keypath": "results"
+                          },
+                          "signature": "84a046d923853d8b75498a95335a2256a9f14b1949ef124e17ac432df7bce61f8dfe0b8d49a17bc2ac6b2fba3cd7ec70+1624032394782"
```
[the remark this answers is not in the document]
Comment (vinbab) [reply dashboards/ds-solarwinds-webapi/dashboard.json:94]: @adamkinniburgh do I just remove the line and the comma before?

File: dashboards/ds-solarwinds-webapi/dashboard.json [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+              }
+            ],
+            "width": 75.146
+          }
+        ]
+      }
+    ]
+  },
+  "derivedFrom": "00000000-0000-0000-0000-000000000000",
+  "draft": true,
```
Comment (Sameer-Mhaisekar) [inline dashboards/ds-solarwinds-webapi/dashboard.json:413]: I usually delete the draft line. Also I think this means this dashboard will be imported as a draft unpublished. Maybe @adamkinniburgh can confirm

File: dashboards/ds-solarwinds-webapi/dashboard.json [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+  "openAccess": {
```
Comment (Sameer-Mhaisekar) [inline dashboards/ds-solarwinds-webapi/dashboard.json:414]: Also not sure if we publish dashboards with OA enabled by default.

Comment (jellyash) [reply dashboards/ds-solarwinds-webapi/dashboard.json:414]: This should not be enabled by default. Providing unauthenticated access to SolarWinds data would add our nails to their coffin.

File: dashboards/ds-solarwinds-webapi/readme.md (hunk 1 above)
Comment (jellyash) [reply dashboards/ds-solarwinds-webapi/readme.md]: Do we want to detail the provider side of this at all? We skipped this during our discussion on Friday.

File: dashboards/ds-solarwinds-webapi/readme.md (hunk 2 above)
Comment (jellyash) [reply dashboards/ds-solarwinds-webapi/readme.md:10]: Do we want to detail the provider side of this at all? We skipped this during our discussion on Friday.

File: dashboards/ds-solarwinds-webapi/dashboard.json (hunk 4 above)
Comment (adamkinniburgh) [reply dashboards/ds-solarwinds-webapi/dashboard.json:413]: Disasterous for dashboard packs, but in this case it shouldn't hurt as once you've pasted in the JSON and clicked apply, you'd then click Publish regardless, and the `draft: true` would be dropped to false as part of that.

File: dashboards/ds-solarwinds-webapi/dashboard.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamkinniburgh) [reply dashboards/ds-solarwinds-webapi/dashboard.json:94]: > @adamkinniburgh do I just remove the line and the comma before?

Yep. If if the last line in tile then remove "signature" and the comma from the line before.

If you ever want to check your JSON is still valid, this is my go-to tool... https://jsonlint.com

File: dashboards/ds-solarwinds-webapi/readme.md (hunk 1 above)
Comment (jellyash) [reply dashboards/ds-solarwinds-webapi/readme.md]: Does it make sense to have the images folder as its own folder with a readme we can link to?

File: dashboards/ds-solarwinds-webapi/dashboard.json (hunk 5 above)
Comment (vinbab) [reply dashboards/ds-solarwinds-webapi/dashboard.json:409]: Agree. That has already been set to false on a recent commit. Thanks.

**Comment by adamkinniburgh** [comment]
@vinbab SO CLOSE... you've called the folder `ds-` but this is a SCOM dashboard. We decided to drop the product name from the folder name anyway, so just call the folder `solarwinds-orion-npm` to reflect what it's for, then call the file `scom.json`. Once we merge this one in, we can quickly spin out `ds.json` and `azure.json`

**Review by Sameer-Mhaisekar — APPROVED** [review]

**Pull request merged**